### PR TITLE
chore: WDS Color `bgAccentHover` and `bgAccentActive` (light mode)

### LINKED ISSUE #22713: Refine `bgAccentHover` and `bgAccentActive` (light mode)
Should take into account the lightness of the seed — currently not visible for darkest seeds and too harsh in light seeds.

### PR DESCRIPTION
**Pull request by ichik**
## Description

tl;dr Gradated hover and active manipulation (more for darker colors, less for lighter), inverted rules for very light seeds

Fixes #22713

## Media
New on the left, current on the right
https://user-images.githubusercontent.com/80973/234889970-05792d7c-e5c7-49b0-80a2-6c9b31d2751d.mov

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Chore (housekeeping or task changes that don't impact user perception)

## How Has This Been Tested?

- Manual

### Test Plan
Initial POC refinement, no testing necessary

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

**Comment by ichik** [comment]
> Subjectively, but it seems that Accent Active turned out to be quite dark. And what about the dark mode, is everything okay there?

To me seemed to be the minimum that'd be perceptually noticeable. I'll do another round of evaluation.

Dark mode will be iterated upon separately.
